judoscale-ruby/rails-autoscale-core.gemspec

```diff
@@ -8,7 +8,7 @@ Gem::Specification.new do |spec|
   spec.authors = ["Adam McCrea", "Carlos Antonio da Silva", "Jon Sullivan"]
   spec.email = ["hello@judoscale.com"]
 
-  spec.summary = "This gem works with the Judoscale Heroku add-on to automatically scale your web and worker dynos."
+  spec.summary = "Autoscaling for Ruby."
```

Comment:
Just wasn't sure if needed to add more, since it's the "core" gem that others depend on, so kept it simple. LMK if you want to expand it further.